code/datums/elements/footstep.dm

```diff
@@ -134,21 +134,31 @@
 	if(!source_loc)
 		return
 
-	play_fov_effect(source, 5, "footstep", direction, ignore_self = TRUE)
+	//cache for sanic speed (lists are references anyways)
+	var/static/list/footstep_sounds = GLOB.footstep
```

Comment:
really? that bad?

Reply:
in retrospect im pretty sure this is almost useless since it now has to go from accessing a global datum and then a field of that datum to just accessing a global field. since its used three times it would be faster to make it a local reference

Reply:
the fuck